src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -118,6 +118,7 @@ public partial class HttpClientHandler : System.Net.Http.HttpMessageHandler
         public System.Net.CookieContainer CookieContainer { get { throw null; } set { } }
         [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
         public System.Net.ICredentials? Credentials { get { throw null; } set { } }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
```

Comment:
What about DangerousAcceptAnyServerCertificateValidator is unsupported?  ServerCertificateCustomValidationCallback is unsupported, but this delegate is just a singleton that always returns true.

Reply:
OK, will annotate that one too

Reply:
> OK, will annotate that one too

Why "too"?  I'm wondering what about DangerousAcceptAnyServerCertificateValidator required it to be annotated at all?

Reply:
Oh sorry misunderstood, i think its type parameter `X509Certificate2` was unsupported `System.Func<System.Net.Http.HttpRequestMessage, System.Security.Cryptography.X509Certificates.X509Certificate2?, System.Security.Cryptography.X509Certificates.X509Chain?, System.Net.Security.SslPolicyErrors, bool> DangerousAcceptAnyServerCertificateValidator`

Reply:
> i think its parameter X509Certificate2 was unsupported

Why does a parameter to a delegate trigger the analyzer?

Reply:
> when the delegate isn't even being invoked

It is used here https://github.com/dotnet/runtime/pull/43363/files#diff-834c6afe6e91f0098d9e2cdd5e0ae143de990617f321e9d0e98045f802d0a14fR311-R312

Reply:
@stephentoub  should we suppress it instead?